app/src/test/kotlin/fr/free/nrw/commons/TestCommonsApplication.kt

```diff
@@ -1,6 +1,5 @@
 package fr.free.nrw.commons
```

Comment:
Reverting the changes in this file to the way they were in https://github.com/commons-app/apps-android-commons/commit/fa87eb566128c777dde3afc79ba2b68ce70efd55 seems to fix the CI issue. I couldn't see why this is the case. Can anyone shed some light about this?

Reply:
cc: @ashishkumar468